.buildkite/commands/run-unit-tests-reader.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash -eu
+
+# Run this at the start to fail early if value not available
```

I'm not sure that would actually fail early if value not available, because despite this script having `-eu` in the shebang, if that script is `source`'d then the shebang is ignored…

If you really want such behavior you might want to extract the `-eu` from the shebang into dedicated `set` instructions instead